desktop/src/main/java/bisq/desktop/main/dao/governance/GovernanceView.java

```diff
@@ -142,7 +136,9 @@ protected void activate() {
     @SuppressWarnings("Duplicates")
     @Override
     protected void deactivate() {
-        daoFacade.phaseProperty().removeListener(phaseChangeListener);
+        if (phaseChangeSubscription != null)
+            phaseChangeSubscription.unsubscribe();
```

Comment:
As we had issues in the past concerning using no brackets for one line conditionals we decided to use always {}

Reply:
Fixed